adapters/socketio_berkeley.c

```diff
@@ -536,31 +536,31 @@ static int initiate_socket_connection(SOCKET_IO_INSTANCE* socket_io_instance)
         }
     }
 
-    if (socket_io_instance->address_type == ADDRESS_TYPE_IP)
-    {
-        socket_io_instance->socket = socket(addr->ai_family, addr->ai_socktype, addr->ai_protocol);
-    }
-    else
+    if(result == 0)
```

Comment:
It would be great if we could add a UT as well.

Reply:
The UT for this module are disabled. Filed an issue to track that.
https://github.com/Azure/azure-c-shared-utility/issues/638